doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
-The downside of para-virtualization interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+The next step after the NFVI Reference Model has been identified and developed is to take the general model, which is purposely designed to be able to be applied to a number of technologies, and apply it to a discrete number of concrete and ultimately deployable Reference Architecture platforms. The intention is to chose the reference architectures carefully so that there will only be a small set of architectures that meets the specific requirements for supporting NFV and Telecom specific applications. Per the principles laid out in the Reference Model documentation, the Reference Architectures need to meet the following criteria as much as is practical:
 
-<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 4:</b> Para-Virtualized interface components (software only).</p>
+  - Initially should be based on widely established technology and systems used in the Telecom Industry.  This will help ensure a faster adoption rate because the operators are already familiar with the technology and might even have systems in production. Another advantage to this approach is a project faster development cycle.
+  - Subsequent architectures should be based on either additional established or promising emerging technologies that are chosen by the community members.  
 
-<a name="4.1.2"></a>
-#### 4.1.2 Direct assignment with IOMMU.
+<a name="1.4"></a>
+## 1.4 Common NFVI Benefits
+By providing a pre-defined environment with common capabilities, applications are able to be developed and deployed more rapidly.  In addition, the common infrastructure can be optimized for various workloads, such as IT (Information Technology), VNF, AI (Artificial Intelligence), and other future workload types as new technologies emerge. The benefits of this approach are:
 
-Direct Assignment is supported in x86 architectures through an IOMMU (Input/Ouput Memory Management Unit), which provides the ability for a PCIe device to autonomously (i.e. without hypervisor intervention) perform DMA transfers directly into guest memory as shown in **Figure 5**.
+- Configuration automation over customization
+  - By abstracting the infrastructure capabilities as much as possible, operators are able to use common infrastructure platforms across all VNF vendors.
+  - Maintaining a consistent infrastructure allows for higher levels of automation due to a reduced need for customization of the various components.
```

Comment:
customization -> customisation

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/ccd069304d5bb56d554fd8aaf2a8f02c8baa875e